src/fetch-wrapper.ts

```diff
@@ -39,6 +39,7 @@ export default function fetchWrapper(
         body: requestOptions.body,
         headers: requestOptions.headers as HeadersInit,
         redirect: requestOptions.redirect,
+        ...(requestOptions.body && { duplex: 'half' })
```

Comment:
we should add a comment about why the `duplex` option is added here, with a URL that holds more information. Will `half` always work, or are there different values that need to be set based on different types of request body?

Reply:
To my understanding, based on [the spec](https://fetch.spec.whatwg.org/#enumdef-requestduplex) right now half should be appropriate to set for all cases in Node.js (it is the also only option available at the moment)